airbyte-integrations/connectors/source-retently/acceptance-test-config.yml

```diff
@@ -4,28 +4,23 @@ connector_image: airbyte/source-retently:dev
 acceptance_tests:
   spec:
     tests:
-      - spec_path: "source_retently/spec.json"
+      - spec_path: "source_retently/spec.yaml"
   connection:
     tests:
       - config_path: "secrets/config.json"
         status: "succeed"
-      - config_path: "secrets/old_config.json"
-        status: "succeed"
-      - config_path: "secrets/config_oauth.json"
-        status: "succeed"
       - config_path: "integration_tests/invalid_config.json"
         status: "failed"
   discovery:
     tests:
       - config_path: "secrets/config.json"
-        backward_compatibility_tests_config:
-          disable_for_version: "0.1.5"
```

Comment:
Some of the changes you've made to some of the schema files is resulting in backwards incompatibility. Can you disable these tests for the prior version?